libs/vertexai/tests/integration_tests/vectorstores/feature_store/test_feature_store_bf_executor.py

```diff
@@ -0,0 +1,113 @@
+"""
+Test Vertex Feature Store Vector Search with BF local executor.
+"""
+
+import os
+import uuid
+
+import pytest
+
+from langchain_google_vertexai.vectorstores.feature_store.executors import (
+    BruteForceExecutor,
+)
+from langchain_google_vertexai.vectorstores.feature_store.feature_store import (
+    VertexFeatureStore,
+)
+
+TEST_TABLE_NAME = "langchain_test_table"
+
+
+@pytest.fixture(scope="class")
+def store_bf_executor(request: pytest.FixtureRequest) -> VertexFeatureStore:
+    """BigQueryVectorStore tests context.
+
+    In order to run this test, you define PROJECT_ID environment variable
+    with GCP project id.
+
+    Example:
+    export PROJECT_ID=...
+    """
+    from google.cloud import bigquery
+
+    from langchain_google_vertexai import VertexAIEmbeddings
+
+    embedding_model = VertexAIEmbeddings(
+        model_name="textembedding-gecko@latest",
+        project=os.environ.get("PROJECT_ID", None),
+    )
+    TestVertexFeatureStore_bf_executor.store_bf_executor = VertexFeatureStore(
+        project_id=os.environ.get("PROJECT_ID", None),  # type: ignore[arg-type]
+        embedding=embedding_model,
+        location="us-central1",
+        dataset_name=TestVertexFeatureStore_bf_executor.dataset_name,
+        table_name=TEST_TABLE_NAME,
+        executor=BruteForceExecutor(),
+    )
+    TestVertexFeatureStore_bf_executor.store_bf_executor.add_texts(
+        TestVertexFeatureStore_bf_executor.texts,
+        TestVertexFeatureStore_bf_executor.metadatas,
+    )
+
+    def teardown() -> None:
+        bigquery.Client(location="us-central1").delete_dataset(
+            TestVertexFeatureStore_bf_executor.dataset_name,
+            delete_contents=True,
+            not_found_ok=True,
+        )
+
+    request.addfinalizer(teardown)
+    return TestVertexFeatureStore_bf_executor.store_bf_executor
+
+
+class TestVertexFeatureStore_bf_executor:
+    """BigQueryVectorStore tests class."""
+
+    dataset_name = uuid.uuid4().hex
+    store_bf_executor: VertexFeatureStore
+    texts = ["apple", "ice cream", "Saturn", "candy", "banana"]
+    metadatas = [
+        {
+            "kind": "fruit",
+        },
+        {
+            "kind": "treat",
+        },
+        {
+            "kind": "planet",
+        },
+        {
+            "kind": "treat",
+        },
+        {
+            "kind": "fruit",
+        },
+    ]
+
+    @pytest.mark.skipif(os.getenv("RUN_SKIPPED") != "1", reason="CI/CD not ready.")
```

Comment:
just mark these tests as `extended`, please, and add missing infra to our internal test project

Reply:
done